src/test/kotlin/org/jetbrains/research/transformations/AnonymizationTest.kt

```diff
@@ -0,0 +1,62 @@
+package org.jetbrains.research.transformations
```

Comment:
I think it is ok because the PR to the develop branch. I think it would be great to finish both transformations with tests in the develop branch. This process can be iteratively

Reply:
Ok, got it